### PR TITLE
New package: gopm

### PR DESCRIPTION
**Pull request by jprjr**

### CODE REVIEW

**Comment by thypon** [comment]
Please rebase on top of master. There is a problem with go build_style without.

**Comment by jprjr** [comment]
@thypon rebased, fixed, and pushed!

**Comment by thypon** [comment]
Almost done: now rename commit 'New package: gopm-git' to 'New package: gopm' and remove go_get line.

**Comment by jprjr** [comment]
Done and done

**Comment by jprjr** [comment]
Changed the order of version/revision

**Comment by thypon** [comment]
Just to know: the stable version isn't functional?

**Comment by jprjr** [comment]
The last released version is from about a year ago, I can't get it to compile

**Comment by thypon** [comment]
btw a version in the form $laststableversion.$dateoflastcommit should also rule out update-check related errors. Maybe it's better to wait for @xtraeme opinion.

@dominikh could you include a `: ${go_cmd:=go}` instead of plain go to made possible to switch to gopm?

File: srcpkgs/gopm/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template build file for 'gopm'
+pkgname=gopm
+version=20150713
```
Comment (xtraeme) [inline srcpkgs/gopm/template]: Please version it as `0.0.date`, this way we can upgrade when an official version is released.

**Comment by jprjr** [comment]
Hi sorry, I've been out of town for about a week - expect some fixes soon!

**Comment by jprjr** [comment]
Fixed and rebased

**Pull request merged**